### ISSUE TITLE
Restrucutre and use Postgres DB

### ISSUE THREAD

**Issue by bridgesign** [bridgesign/EmailParser#1]
The current DB is the default SQLite that comes with django. We require to shift to PostgreSQL. Moreover, the current storage is trivial and further distinctions like whether it is seminar, or workshop, open to all or does it require registration and the registration link.

**Comment by SoumyaRanjanNaik** [comment]
Can I get work on this issue ?

**Comment by bridgesign** [comment]
Yes. We can start with converting to postgres. We also need to think of all different fields that can come with a workshop or seminar. Current implementation cannot distinguish them, but it will be good to add the columns for these things.
